.eslintignore

```diff
@@ -0,0 +1,4 @@
+test/*
+public/*
+coverage/*
+node_modules/*
```

Comment:
I didn't put all the things, because I didn't know if the proposal would be accepted